blocks/article-header/article-header.css

```diff
@@ -36,34 +41,33 @@
       }
     }
 
-    & :is(h3):not(.author-meta__name) {
+    & h3:not(.author-meta__name) {
```

Comment:
These couple `:is()` selectors with the single heading element within them had the same specificity and weren't needed.